db/commitlog/commitlog.cc

```diff
@@ -904,41 +946,77 @@ class db::commitlog::segment : public enable_shared_from_this<segment>, public c
         }
 
         if (_closed) {
-            return make_exception_future<rp_handle>(std::runtime_error("commitlog: Cannot add data to a closed segment"));
+            return make_exception_future<>(std::runtime_error("commitlog: Cannot add data to a closed segment"));
         }
 
         buf_memory -= permit.release();
         _segment_manager->account_memory_usage(buf_memory);
 
-        replay_position rp(_desc.id, position());
-        _cf_dirty[id]++; // increase use count for cf.
+        auto& out = _buffer_ostream;
+
+        std::optional<crc32_nbo> mecrc;
+
+        // if this is multi-entry write, we need to add an extra header + crc
+        // the header and crc formula is:
+        // header:
```

Comment:
A little bird told me to pass to you that this should be in the docs/ as well, not only comments.

Reply:
You are hearing sounds?!

Reply:
Constantly! From Google (TM) Meet.

Reply:
So, terrible question: Docs where? There is no commit log entry in the MD docs in the source tree. There is a very old and not very informative (nothing on file format) wiki page, but that seems lame. 
I would be happy to add a secondary issue to actually create a commit log format and mechanism(?) document later, outside this PR.

Reply:
In docs/, ok for adding it later.